### ISSUE TITLE
Blender Import Issue

### ISSUE THREAD

**Issue by dtupper** [cubedparadox/Cubeds-Unity-Shaders#2]
When you import a model into Blender using MMD_tools, then export the model from Blender into FBX, the cell shader offset texture looks quite odd:

http://i.imgur.com/XYa2qTD.png

In this image I only applied the shader to the model's pupil/iris materials. As you can see, the textures project far too far out. This doesn't occur with ToonShader, for example, but does occur with some other toon shaders (Aubergine's Toon Shaders)

**Comment by dtupper** [comment]
As an aside I believe it may have something to do with model scaling upon import. I will test it this afternoon and post results.

**Comment by cubedparadox** [comment]
Yeah, this is a scaling issue, I can see if I can automatically correct for it at some point, in the meanwhile changing the model import scale by a factor of 100/0.01 normally solves the problem for me.

**Comment by dtupper** [comment]
Thanks for taking a look.When you say "model import scale" do you mean the PMX -> Blender import process, or the Blender -> FBX export?

**Comment by cubedparadox** [comment]
I mean the fbx -> unity import, there's an import scale setting in the inspector for the model

**Comment by dtupper** [comment]
Perfect. That worked. In the prefab asset inspector, I set the scale to 100, and had to scale down the model massively on the prefab in the scene (in my case, 0.0005)

Thanks again!
